### ISSUE TITLE
[Bug Report] Display language change doesn't work properly

### ISSUE THREAD

**Issue by IoeCmcomc** [Amulet-Team/Amulet-Map-Editor#855]
### Current Behaviour:
Changing the language only updates the menu bar when I switched to another tab. It also doesn't update labels in currently opened 3D editors. Labels in the vertical tab bar aren't updated at all. Only worlds opened **after** the language change show the current language correctly.

### Expected behaviour:
Switching the language should update all displayed texts to the changed language.

### Steps To Reproduce:
(Assume the startup language is English)
1. Open a world, but don't switch to the "3D Editor" tab yet.
2. Change the language to another one (i.e. Japanese).
3. The menu bar does not update immediately.
4. Switch again to the opened world.
5. _The display language in the "About" and "Convert" tabs is still English._
6. Switch to the "3D Editor" tab.
7. Change the language to another one (i.e. French).
8. Switch again to the opened world.
9. _The language in the 3D editor (tooltips included) is Japanese, and the vertical tab bar is in English._

### Environment:
- OS: Windows 11 Home (22H2)
- Minecraft Platform: Java
- Minecraft Version: any
- Amulet Version: 0.10.7

### Additional context
There are no warnings or errors in the console.

### Screenshots
![amulet_app_AkJvb0gs9r](https://user-images.githubusercontent.com/53734763/209129815-c04eefef-42b5-4f48-9cac-b6098a889d7d.png)

**Comment by gentlegiantJGC** [comment]
Yes the language selection only applies to newly created UI elements.

I might look into fixing this in the new version.

For now you will need to restart the application for it to apply correctly.
